src/EnergyPlus/ZoneTempPredictorCorrector.hh

```diff
@@ -226,6 +247,12 @@ namespace ZoneTempPredictorCorrector {
 	CalcZoneAirTempSetPoints();
 
 	void
+	CalculateMonthlyRunningAverageDryBulb( Array1D< Real64 > & runningAverageASH, Array1D< Real64 > & runningAverageCEN );
```

Comment:
This function aims to calculate the value of the **runningAverageASH** and **runningAverageCEN** , so the pointer sign is to call by reference. The next one only uses the value of the two arrays, and I'd assume it's better only to call by value so the array won't be messed up in this function. Not sure if necessary or appropriate here.